tests/e2e/tests/devfiles/JavaMaven.spec.ts

```diff
@@ -8,147 +8,33 @@
-
-            await ide.waitNotification('has exited with code 0.', 120000);
-        });
-
-        test('Close the terminal tasks', async () => {
-            await terminal.closeTerminalTab('maven build and run');
-        });
+        commonTestMethods.runTaskAndCloseTerminal('maven build', 120000);
+        commonTestMethods.runTaskAndCloseTerminal('maven build and run', 120000);
     });
 
     suite('Language server validation', async () => {
-        test('Expand project and open file in editor', async () => {
-            await projectTree.expandPathAndOpenFileInAssociatedWorkspace(fileFolderPath, tabTitle);
-            await editor.selectTab(tabTitle);
-        });
-
-        test('Java LS initialization', async () => {
-            await ide.waitStatusBarContains('Starting Java Language Server', 20000);
-            await ide.waitStatusBarTextAbsence('Starting Java Language Server', 1800000);
-            await ide.waitStatusBarTextAbsence('Building workspace', 360000);
-        });
-
-        test('Suggestion invoking', async () => {
-            await ide.closeAllNotifications();
-            await editor.waitEditorAvailable(tabTitle);
-            await editor.clickOnTab(tabTitle);
-            await editor.waitEditorAvailable(tabTitle);
-            await editor.waitTabFocused(tabTitle);
-            await editor.moveCursorToLineAndChar(tabTitle, 10, 20);
-            await editor.pressControlSpaceCombination(tabTitle);
-            await editor.waitSuggestion(tabTitle, 'append(char c) : PrintStream');
-        });
-
-        test('Error highlighting', async () => {
-            await editor.type(tabTitle, 'error', 11);
-            await editor.waitErrorInLine(11);
-            await editor.performKeyCombination(tabTitle, Key.chord(Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE));
-            await editor.waitErrorInLineDisappearance(7);
-        });
-
-        test('Autocomplete', async () => {
-            await editor.moveCursorToLineAndChar(tabTitle, 10, 11);
-            await editor.pressControlSpaceCombination(tabTitle);
-            await editor.waitSuggestionContainer();
-            await editor.waitSuggestion(tabTitle, 'System - java.lang');
-        });
-
-        test('Codenavigation', async () => {
-            await editor.moveCursorToLineAndChar(tabTitle, 9, 10);
-            await editor.performKeyCombination(tabTitle, Key.chord(Key.CONTROL, Key.F12));
-            await editor.waitEditorAvailable(codeNavigationClassName);
-        });
-
-    });
-
-    suite('Stop and remove workspace', async () => {
-        test('Stop workspace', async () => {
-            await dashboard.stopWorkspaceByUI(workspaceName);
-        });
-
-        test('Delete workspace', async () => {
-            await dashboard.deleteWorkspaceByUI(workspaceName);
-        });
+        commonTestMethods.openFileInAssociatedWorkspace(fileFolderPath, tabTitle);
```

Comment:
It looks like you use outdated version of tests, because we get rid of this switchin for a long time ago.

Reply:
So maybe this method should be removed then so no one will use that. I'll do that in a new commit in this PR. What method should I use instead?

Reply:
Nothing, just open workspace and do all actions as a user.